components/Consultation.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <v-col cols="12" md="6">
+    <lazy-static-info
+      class="Consultation"
+      :url="'https://www.fukushihoken.metro.tokyo.lg.jp/iryo/kansen/coronasodan.html'"
```

Comment:
静的な値なのでv-bindを使わずに直接記述できるかと思います。

```suggestion
      url="https://www.fukushihoken.metro.tokyo.lg.jp/iryo/kansen/coronasodan.html"
```

cf. https://jp.vuejs.org/v2/guide/components-props.html#%E9%9D%99%E7%9A%84%E3%81%82%E3%82%8B%E3%81%84%E3%81%AF%E5%8B%95%E7%9A%84%E3%81%AA%E3%83%97%E3%83%AD%E3%83%91%E3%83%86%E3%82%A3%E3%81%AE%E5%8F%97%E3%81%91%E6%B8%A1%E3%81%97